velox/dwio/parquet/reader/CMakeLists.txt

```diff
@@ -35,3 +35,8 @@ target_link_libraries(
   parquet
   thrift
   ${FMT})
+
+if(${VELOX_ENABLE_QPL})
+  target_sources(velox_dwio_native_parquet_reader PUBLIC CompressionQpl.cpp)
+  target_link_libraries(velox_dwio_native_parquet_reader qpl)
```

Comment:
IIUC this PR only modifies the internals of the parquet reader in case qpl is activated right? So qpl should be linked privately as a build requirement?
```suggestion
  target_sources(velox_dwio_native_parquet_reader PRIVATE CompressionQpl.cpp)
  target_link_libraries(velox_dwio_native_parquet_reader PRIVATE qpl)
```

Reply:
Thanks for your suggestion, but maybe it can`t set private in target_link_libraries statement due to conflict with over. I will modify PUBLIC to PRIVATE in target_sources statement in the later commit.

Reply:
>  due to conflict with over

I am not sure I understand what you mean?

Reply:
![image](https://user-images.githubusercontent.com/46258124/236385984-05250abf-007d-4b3a-b634-897b4e940e52.png)

it means that, if i set PRIVATE keyword in target_link_libraries statement, it will report this error when building.

Reply:
@assignUser If you have any idea about this, it will be helpful.

Reply:
Ah I see. That's not something you can fix. :+1: